src/components/button/Styles.scss

```diff
@@ -14,4 +19,14 @@
   &--inactive{
     background-color: $elephant-paw;
   }
+}
+
+.submitButton div {
+  padding: 4px;
+}
+
+#submitAnswerButton img {
+  width: 20px;
```

Comment:
See comment above and delete this id and move the rules on the image into the class above.